templates/zerver/api/changelog.md

```diff
@@ -1028,17 +1028,24 @@ No changes; feature level used for Zulip 3.0 release.
 * [`GET /streams`](/api/get-streams): Replaced
   `is_announcement_only` boolean with `stream_post_policy` enum for
   specifying who can post to a stream.
-* `GET /api/v1/user_uploads`: Added new endpoint for requesting a
-  temporary URL for an uploaded file that does not require
-  authentication to access (e.g. for passing from a Zulip desktop,
-  mobile, or terminal app to the user's default browser).
-* Added `EMAIL_ADDRESS_VISIBILITY_NOBODY` possible value for
+* [`GET /user_uploads/{realm_id_str}/{filename}`](/api/get-file-temporary-url):
+  Added new endpoint for requesting a temporary URL for an uploaded
+  file that does not require authentication to access (e.g. for passing
+  from a Zulip desktop, mobile, or terminal app to the user's default
+  browser).
+* [`GET /events`](/api/get-events), [`POST /register`](/api/register-queue):
+  Added `EMAIL_ADDRESS_VISIBILITY_NOBODY` as a possible value for
   `email_address_visibility`.
-* Added `private_message_policy` realm setting.
-* `muted_topic` objects now are a 3-item tuple: (`stream_id`, `topic`,
-  `date_muted`).  Previously, they were a 2-item tuple.
-* `GitLab` authentication is now available.
-* Added `None` as a video call provider option.
+* [`GET /events`](/api/get-events), [`POST /register`](/api/register-queue):
+  Added `private_message_policy` realm setting.
+* [`GET /events`](/api/get-events), [`POST /register`](/api/register-queue):
+  `muted_topics` array objects now are 3-item tuples: (`stream_name`,
+  `topic_name`, `last_updated`). Previously, they were 2-item tuples and
+  did not include the `last_updated` item.
+* [`GET /server_settings`](/api/get-server-settings): `GitLab`
+  authentication is now available.
```

Comment:
nit: should be "GitLab authentication" or "`gitlab` authentication" (or some further rewording.)

That is, code font should be reserved for strings that literally appear in the API as identifiers, and for other forms of literal code. Here, `gitlab` is such an identifier because what appears in the API is something like `"gitlab": true` or `"gitlab": false`.